tests/test_tools.py

```diff
@@ -1748,8 +1748,46 @@ def test_is_fully_appointed(appts, start, end, expected):
         ),
     ],
 )
-def test_get_id_from_name(input, expected):
-    assert awards_grants_honors(input) == expected
+def test_awards_grants_honors(input, expected):
+    assert awards_grants_honors(input, "service") == expected
+
+
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        (
+            {
+                "funding": [
+                    {"name": "Omega Laser User's Group Travel Award", "value": 1100, "year": 2013},
+                    {"name": "NIF User's Group Travel Award", "value": 1150, "year": 2013},
+                ],
+                "honors": [
+                    {"name": "CVPR best paper award", "year": 2024, "month": 5},
+                    {"name": "ICLR paper accepted", "year": 2022, "month": 3},
+                ]
+            },
+            [
+                {"description": "CVPR best paper award", "year": 2024, "_key": 2024.05},
+                {"description": "ICLR paper accepted", "year": 2022, "_key": 2022.03},
+                {"description": "Omega Laser User's Group Travel Award (\\$1,100)", "year": 2013, "_key": 2013.0},
+                {"description": "NIF User's Group Travel Award (\\$1,150)", "year": 2013, "_key": 2013.0},
+
+            ],
+        ),
+        (
+            {
+                "funding": [{"name": "Omega Laser User's Group Travel Award", "value": 1100, "year": 2013}],
+                "honors": [{"name": "CVPR best paper award", "year": 2024, "month": 5}]
+            },
+            [
+                {"description": "CVPR best paper award", "year": 2024, "_key": 2024.05},
+                {"description": "Omega Laser User's Group Travel Award (\\$1,100)", "year": 2013, "_key": 2013.0},
+            ],
+        ),
+    ],
+)
+def test_honors_grants_honors_t2(input, expected):
```

Comment:
rather than write a different test function for this, can we do it by varying the input?

I think the best struture for this would be to define an input that contains funding, honors and service etc., and pass it to the function but then give as input in the parameterize the allowed tag (`service`,`honors`, etc.) and have it return the right thing.